package.json

```diff
@@ -22,12 +22,7 @@
     "husky": "^2.4.0",
     "lerna": "^3.15.0",
     "prettier": "^1.18.2",
-    "typescript": "^3.4.3"
-  },
-  "resolutions": {
-    "lodash": ">=4.17.12",
-    "mixin-deep": ">=1.3.2",
-    "set-value": ">=3.0.1"
```

Comment:
resolutions only really seem to help for local development. fresh projects using web-scripts would need to set their own resolutions to get lodash in the right place to get postinstall to pass. Not optimal! We should avoid having resolutions from here on out. I may even explore setting up a rule.